pyproject.toml

```diff
@@ -259,11 +241,8 @@ commands =
 [testenv:lint]
 description = Check the code style
 commands =
-  black --check --diff --color .
-  isort --check --diff --color .
-  flake8 --max-line-length=165 --config=/dev/null anta tests
-  pylint anta
-  pylint tests
+  pylint --rcfile=pylintrc anta
```

Comment:
probably need to add ruff here eventually

Reply:
Yes, when the 600 errors are addressed

Reply:
Done!